pythonforandroid/toolchain.py

```diff
@@ -1477,15 +1601,30 @@ def extract_source(self, source, cwd):
     #         print("Unrecognized extension for {}".format(filename))
     #         raise Exception()
 
-    def apply_patch(self, filename):
+    def apply_patch(self, filename, arch='armeabi'):
         """
         Apply a patch from the current recipe directory into the current
         build directory.
         """
         info("Applying patch {}".format(filename))
         filename = join(self.recipe_dir, filename)
-        # AND: get_build_dir shouldn't need to hardcode armeabi
-        sh.patch("-t", "-d", self.get_build_dir('armeabi'), "-p1", "-i", filename)
+        shprint(sh.patch, "-t", "-d", self.get_build_dir(arch), "-p1",
+                   "-i", filename, _tail=10)
+
+    def apply_all_patches(self, wildcard=join('patches','*.patch'), arch='armeabi'):
```

Comment:
I understand the idea of this one, but I'm not sure about it - it could lead to fragile situations where things break if a new patch is added in the directory (e.g. during debug), and also only supports using the recipe dir. How would you feel about just dropping this function?

I can remove it on merge if you agree, there's no need for you to do it yourself if you don't want to mess with it.